src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
+    private int randomIndex = randomColor.nextInt(Color.values().length);

```

Comment:
the mentor meant to make this change in a method, not as a class field. Because in this case you will always have the same index